chia/util/chia_logging.py

```diff
@@ -38,56 +40,59 @@ def get_file_log_handler(
+    log_level = logging_config.get("log_level", default_log_level)
     file_name_length = 33 - len(service_name)
     log_date_format = "%Y-%m-%dT%H:%M:%S"
     file_log_formatter = logging.Formatter(
         fmt=f"%(asctime)s.%(msecs)03d {service_name} %(name)-{file_name_length}s: %(levelname)-8s %(message)s",
         datefmt=log_date_format,
     )
+    handlers: List[logging.Handler] = []
     if logging_config["log_stdout"]:
-        handler = colorlog.StreamHandler()
-        handler.setFormatter(
+        stdout_handler = colorlog.StreamHandler()
+        stdout_handler.setFormatter(
             colorlog.ColoredFormatter(
                 f"%(asctime)s.%(msecs)03d {service_name} %(name)-{file_name_length}s: "
                 f"%(log_color)s%(levelname)-8s%(reset)s %(message)s",
                 datefmt=log_date_format,
                 reset=True,
             )
         )
-
-        logger = colorlog.getLogger()
-        logger.addHandler(handler)
+        handlers.append(stdout_handler)
     else:
-        logger = logging.getLogger()
-        logger.addHandler(get_file_log_handler(file_log_formatter, root_path, logging_config))
+        handlers.append(get_file_log_handler(file_log_formatter, root_path, logging_config))
 
     if logging_config.get("log_syslog", False):
         log_syslog_host = logging_config.get("log_syslog_host", "localhost")
         log_syslog_port = logging_config.get("log_syslog_port", 514)
         log_syslog_handler = SysLogHandler(address=(log_syslog_host, log_syslog_port))
         log_syslog_handler.setFormatter(logging.Formatter(fmt=f"{service_name} %(message)s", datefmt=log_date_format))
-        logger = logging.getLogger()
-        logger.addHandler(log_syslog_handler)
-
-    if "log_level" in logging_config:
-        if logging_config["log_level"] == "CRITICAL":
-            logger.setLevel(logging.CRITICAL)
-        elif logging_config["log_level"] == "ERROR":
-            logger.setLevel(logging.ERROR)
-        elif logging_config["log_level"] == "WARNING":
-            logger.setLevel(logging.WARNING)
-        elif logging_config["log_level"] == "INFO":
-            logger.setLevel(logging.INFO)
-        elif logging_config["log_level"] == "DEBUG":
-            logger.setLevel(logging.DEBUG)
-            logging.getLogger("aiosqlite").setLevel(logging.INFO)  # Too much logging on debug level
-        else:
-            logger.setLevel(logging.INFO)
-    else:
-        logger.setLevel(logging.INFO)
+        handlers.append(log_syslog_handler)
 
     if beta_root_path is not None:
-        logger.addHandler(get_file_log_handler(file_log_formatter, beta_root_path, get_beta_logging_config()))
+        handlers.append(get_file_log_handler(file_log_formatter, beta_root_path, get_beta_logging_config()))
+
+    root_logger = logging.getLogger()
+    log_level_invalid = False
+    for handler in handlers:
+        try:
+            handler.setLevel(log_level)
+        except Exception:
```

Comment:
```
>>> handler.setLevel("lkajdsflk")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/logging/__init__.py", line 916, in setLevel
    self.level = _checkLevel(level)
  File "/usr/lib/python3.8/logging/__init__.py", line 198, in _checkLevel
    raise ValueError("Unknown level: %r" % level)
ValueError: Unknown level: 'lkajdsflk'
```

Let's at least restrict this to handling `ValueError`.  Maybe the message check too.

```suggestion
        except ValueError as e:
            if "unknown level" not in str(e).lower():
                raise
```

Reply:
Hm we then depend on the error message and there is also a `TypeError` btw https://github.com/python/cpython/blob/4652093e1b816b78e9a585d671a807ce66427417/Lib/logging/__init__.py#L208-L218. Initially i had `(ValueError, TypeError)` but went back to `Exception` to not depend on the errors they raise there. Isn't logging an error about the conversion failure enough here if i maybe just add the exception to the log?

Reply:
I suppose this is a corner where not raising, enabling as many loggers as we can, and then logging the error to all of them may be better.  I guess instead of `log_level_invalid` you could just collect a list of exceptions that need reported.

Reply:
> enabling as many loggers as we can, and then logging the error to all of them may be better

Not sure if i follow this, thats what the code currently does, no? (without the exception in the log yet though, which is supposed to be the same for all loggers anyway, no?)

Reply:
Consuming an exception and reporting what you intended it to be is importantly different than reporting what the error actually was.  But sure, not all that different.

Reply:
> Isn't logging an error about the conversion failure enough here if i maybe just add the exception to the log?

I wonder if you read this question ^ in my reply above? :) If so, i don't get the response, but anyway, im adding the exception to the log.

Reply:
The direct answer is yes.  The more confusing reply that includes my various considerations...  is above.

Reply:
Haha, i see. Okay here we go https://github.com/Chia-Network/chia-blockchain/pull/13569/commits/9035286326dde3a5a6ca3c7ba1a40e5a9bb4d7fd. There is no connection between the error and the handler now but.. its anyway always the same log level and with that the same error unless we have some custom handler with special validation in `setLevel` but... 

Example: 

```
2022-09-29T20:24:16.474 harvester root                    : ERROR    Invalid log level found in harvester config: Isdjfo. Defaulting to: WARNING. Errors: Unknown level: 'Isdjfo'
```

Reply:
Okay well, i just added the handler also, now it prints multiple errors but at least there is a connection between the handler and the error 3712a2bebde930cc5d7770c3e0eea6ec5625e994